storybook/main.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+	addons: [
+		'@storybook/addon-docs/react/preset',
```

Comment:
They use `name: '@storybook/addon-docs',` in the migration guide.

Reply:
>Addon-docs configuration gets simpler in 5.3. In 5.2, each framework had its own preset, e.g. @storybook/addon-docs/react/preset. Starting in 5.3, everybody should use @storybook/addon-docs/preset.

Reply:
@epiqueras Thank you! 

I'll make those updates. I'm also not 100% clear if the add-ons need to be Objects to work correctly. It seemed to work as strings (at least for me). I imagined it worked similarly to how Babel's config worked.

Reply:
They can be strings if you don't need extra configuration.